prow/cmd/autobump/bump.sh

```diff
@@ -177,6 +176,18 @@ list() {
   fi
 }
 
+split_on_commas() {
+  local IFS=,
+  local array=($1)
+  echo "${array[@]}"
+}
+
+add_suffix() {
+  local array=($1)
```

Comment:
Do we want this as `"${1[@]}"`?

Reply:
`split_on_commas` is returning a string (because of the `echo`). I don't think `bash` supports an *array* return from a function (without using globals).